models/_sector/nft/trades/ethereum/platforms/blur_v2_ethereum_base_trades.sql

```diff
@@ -0,0 +1,110 @@
+{{ config(
+    schema = 'blur_v2_ethereum',
+    tags = ['dunesql'],
+    alias = alias('base_trades'),
+    partition_by = ['block_date'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_number','tx_hash','sub_tx_trade_id'],
+    )
+}}
+
+{% set blur_v2_start_date = "cast('2023-07-05' as timestamp)" %}
+
+WITH blur_v2_trades AS (
+    SELECT evt_tx_hash AS tx_hash
+    , CAST(ROUND((bitwise_right_shift(collectionPriceSide, 160) - (bitwise_right_shift(collectionPriceSide, 248) * CAST(power(2, 88) AS UINT256)))/POWER(10, 18), 8) AS double) AS price
```

Comment:
```suggestion
    , bytearray_to_uint256(bytearray_substring(cast(collectionPriceSide as varbinary),2,11)) AS price
```
(note this change will be full decimals, so you'd also need to remove the `*POWER(10, 18)` in the final select)

Reply:
done